lxd/instance/drivers/driver_qemu.go

```diff
@@ -3156,8 +3156,11 @@ func (d *qemu) addDriveConfig(bootIndexes map[string]int, driveConf deviceConfig
 
 	var isBlockDev bool
 
-	// Handle local disk devices.
-	if !isRBDImage {
+	// Detect device caches and I/O modes.
+	if isRBDImage {
+		// For RBD, we want writeback to allow for the system-configured "rbd cache" to take effect if present.
+		cacheMode = "writeback"
```

Comment:
So even if we use cacheMode=="writeback" here if the system configured rbd cache isn't enabled then it doesn't use write back mode? Does this introduce any data persistence weaknesses in cases where rbd cache is disabled compared to the current default of not using writeback cacheMode?

Reply:
Ok thanks